docs/data/introduction/licensing/licensing.md

```diff
@@ -56,11 +56,39 @@ Premium package:
 The features exclusive to the Premium version are marked with the <span class="plan-premium"></span> icon throughout the documentation.
 :::
 
+## Upgrading from Community
+
+When you purchase a license for MUI X (Pro or Premium), it will unlock new [packages](#plans) you can use on your project.
+
+If you're using the data grid, the commercial npm packages (`@mui/x-data-grid-pro` and `@mui/x-data-grid-premium`) are a superset of the community.
+
+You can upgrade the dependency on `@mui/x-data-grid` to the respective package of your plan, and replace all your imports.
+
+```diff
+//diff when upgrading to Pro
+
+-import { DataGrid } from '@mui/x-data-grid';
++import { DataGridPro } from '@mui/x-data-grid-pro';
+```
+
+For more details on how to install the packages, please check out our [package installation guide](/x/introduction/installation/).
+
 ## Upgrading from Pro to Premium
 
-You can use your Pro license as a credit when purchasing MUI X Premium.
+When upgrading from Pro, you can upgrade the dependency on `@mui/x-data-grid-pro` to `@mui/x-data-grid-premium` and replace all your data grid imports.
+
+```diff
+//diff when upgrading from Pro to Premium
+
+-import { DataGridPro } from '@mui/x-data-grid-pro';
++import { DataGridPremium } from '@mui/x-data-grid-premium';
+```
+
+:::info
+**Note:** You can use your Pro license as a credit when purchasing MUI X Premium.
```

Comment:
We can cut **Note:** since it's made redundant by the callout.

```suggestion
You can use your Pro license as a credit when purchasing MUI X Premium.
```